<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Ground segmentation -->
  <node name="ground_segmentation" pkg="linefit_ground_segmentation_ros" type="ground_segmentation_node" output="screen">
    <rosparam command="load" file="$(find linefit_ground_segmentation_ros)/launch/segmentation_params.yaml"/>
    <param name="input_topic" value="/kitti/velo/pointcloud" />
    <param name="ground_output_topic" value="ground_cloud" />
    <param name="obstacle_output_topic" value="obstacle_cloud" />
  </node>

</launch>